cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/TransactionHttpHandler.java

```diff
@@ -209,74 +202,72 @@ public void resetTxManagerState(HttpRequest request, HttpResponder responder) {
     responder.sendStatus(HttpResponseStatus.OK);
   }
 
-  @Path("/transactions/pruneinfo")
-  @POST
-  public void getPruneInfo(HttpRequest request, HttpResponder responder) {
+  @Path("/transactions/prune-info/{region-name}")
```

Comment:
I think it would be a good idea to prefix all these REST calls with `prune`, like `/transactions/prune/regions/{region-name}`